spi/transfer-spi/src/main/java/org/eclipse/dataspaceconnector/spi/transfer/provision/ResourceManifestGenerator.java

```diff
@@ -34,20 +28,8 @@
 
     void registerProviderGenerator(ResourceDefinitionGenerator generator);
 
-    void registerPermissionFunctions(Map<String, AtomicConstraintFunction<String, Permission, Boolean>> functions);
-
-    void registerProhibitionFunctions(Map<String, AtomicConstraintFunction<String, Prohibition, Boolean>> functions);
-
-    void registerObligationFunctions(Map<String, AtomicConstraintFunction<String, Duty, Boolean>> functions);
-
     /**
-     * Generates a resource manifest for a data request on a consumer connector. Operations should be idempotent.
+     * Generates a resource manifest for a data request on a connector. Operations should be idempotent.
      */
-    ResourceManifest generateConsumerManifest(TransferProcess process);
-
-    /**
-     * Generates a resource manifest for a data request on a provider connector. Operations should be idempotent.
-     */
-    ResourceManifest generateProviderManifest(TransferProcess process);
-
+    ResourceManifest generateResourceManifest(TransferProcess process);
```

Comment:
Can you add the comments back?

Reply:
The comments are removed because the two connector/provider methods where removed, now the interface offers a single method to generate manifest, and it's documented (line 32)